### ISSUE TITLE
As a consumer, I want to be able to click on a "check out these schools" example

### ISSUE THREAD

**Issue by ErieMeyer** [18F/college-choice#916]
![image](https://cloud.githubusercontent.com/assets/1626026/9569798/fdb44f70-4f43-11e5-90c8-9a5fc521a5ba.png)

These links will be coming from our friends in DPC, but we're blocked until we have 'em.

**Comment by LisaGee** [comment]
dependent on #984

**Comment by meiqimichelle** [comment]
@LisaGee this is separate from #984. Maybe you meant a different issue?

**Comment by LisaGee** [comment]
or #948 ..... Thanks, @meiqimichelle

**Comment by ultrasaurus** [comment]
create a URL like this:  https://ccapi-staging.18f.gov/v1/schools?id=104151,372213

**Comment by shawnbot** [comment]
OK, so as I understand it, the idea here is to link to search URLs like `/search/?id=123,456,etc` that include a list of ids for specific schools. In order for this to work we would need to add a hidden input to the search form:

```html
<input type="hidden" name="id">
```

This *should* just work and would show those schools assuming it's hooked up to an API environment with #948 implemented.

The problem, though, is those school ids would be "sticky", and unless we added something to the UI to explicitly remove them you'd be stuck with that set of schools as the superset to filter by. So I think at the very least we would need to add a link that removes the `id=` parameter. @jjoteal, what do you think?

**Comment by jjoteal** [comment]
@shawnbot this is a tough one, given the timeline we're under. In the interest of not creating a major interruption, perhaps the simplest solution would be to stick the link the blank space right above the school cards in the upper left (on the same row as the page numbers). Perhaps something like this:

![image](https://cloud.githubusercontent.com/assets/12192706/9696857/2a037144-5347-11e5-961e-5105cade7497.png)

Perhaps clicking that would display all schools until search criteria is selected?

Let me know if this idea is too disruptive at this point...

**Comment by LisaGee** [comment]
Dupe of #314 ?

**Comment by ErieMeyer** [comment]
Yep! Closing in favor of https://github.com/18F/college-choice/issues/314
